### ISSUE TITLE
Add "Share on" section

### ISSUE THREAD

**Issue by klokan** [klokantech/www.osmnames.org#2]
The frontpage with map as well as footer in all text pages should have
"Share on" section similar to OldMapsOnline.org

**Comment by sfkeller** [comment]
Good idea to add a "_Share Twitter_(, FB, ...)" button (applies BTW also to OSM2VectorTiles). 

There is an issue though: By including this code, we're feeding G\* & Co. more than needed because every time this page is loaded, G\* & Co. get's a trace. AFAIK no OSM page includes such Like/Share buttons. What about two-clicks-share-buttons jQuery code for social share privacy: http://panzi.github.io/SocialSharePrivacy/  ?

There are other possbile enhancements for such websites (I compiled at https://giswiki.hsr.ch/Best_Practices_for_Web_Maps#Homepage_and_Sections ) like a reference to our _Blogs_ of Geometa Lab HSR (http://geometalab.tumblr.com/tagged/geonames ) and Klokan Technologies (http://blog.klokantech.com/ ?).

And we could also add "_Terms and Conditions_" and "_Credits, Data and Sources_" somewhere. For simplicity we could add these info initially into a static _FAQ_ page though.

**Comment by sfkeller** [comment]
We're having issues with the buttons/icons: They don't show up in the browser. They are probably rendered but seem to be covered by other graphics...
